### ISSUE TITLE
模板继承的问题

### ISSUE THREAD

**Issue by coderlindacheng** [greenlaw110/rythm-doc#40]
![extend](https://cloud.githubusercontent.com/assets/5810023/20091484/fa39ce24-a5cd-11e6-99a3-79cf9830a802.png)
![main](https://cloud.githubusercontent.com/assets/5810023/20091485/fa3fbd2a-a5cd-11e6-87bc-3acbbc35d128.png)

is that work???

**Comment by greenlaw110** [comment]
I don't think it will work as expected.  You have put the `@render(<section-id>)` into a for loop context and refer to the context variable `randObj`. I don't think your `main_rank.html` will even be compiled

**Comment by coderlindacheng** [comment]
我解决了这个问题了...
